sklearn/ensemble/tests/test_voting.py

```diff
@@ -352,6 +463,37 @@ def test_set_params():
     assert_equal(eclf1.get_params()["lr__C"],
                  eclf1.get_params()["lr"].get_params()['C'])
 
+    reg1 = LinearRegression()
+    reg2 = RandomForestRegressor(random_state=1,
+                                 n_estimators=10)
+    reg3 = GradientBoostingRegressor(random_state=1,
+                                     n_estimators=10)
+    ereg1 = AverageRegressor(estimators=[
+        ('lr', reg1), ('rf', reg2)], weights=[1, 2])
+
+    assert_true('lr' in ereg1.named_estimators)
```

Comment:
With the adoption of pytest, we are phasing out use of test helpers `assert_equal`, `assert_true`, etc. Please use bare `assert` statements, e.g. `assert x == y`, `assert not x`, etc.